tests/__init__.py

```diff
@@ -36,7 +36,7 @@ def requireNonAsciiLocale(category_name="LC_ALL"):
 def _decorate_with_locale(category, try_locales, test_method):
     """Decorate test_method to run after switching to a different locale."""
 
-    def _test_under_locale(testself, sftp):
+    def _test_under_locale(testself, *args, **kwargs):
```

Comment:
I think this change might not work under python2.7 (pytest does not seem to apply the correct fixture, maybe a limitation in python2.7's `functools.update_wrapper()`). An alternate fix is to change the call to `test_method()` below to just add the `sftp` arg.